modules/javafx.controls/src/test/java/test/javafx/scene/control/SplitPaneTest.java

```diff
@@ -1329,4 +1335,55 @@ private double convertDividerPostionToAbsolutePostion(double pos, double edge) {
 
         sl.dispose();
     }
+
+    /**
+     * Verifies that a divider position change of the {@link SplitPane} does not hang the layout.
+     * Previously, this may happen when the divider position changed to a large number (>1),
+     * which can hang the layout as it resulted in multiple layout requests (through SplitPaneSkin.layoutChildren).
+     */
+    @Test
```

Comment:
My preference would be to add the bug id to the tests as well ..

Reply:
Added it